src/components/asthma/components/AsthmaActionPlanManager/AsthmaActionPlanManager.css

```diff
@@ -0,0 +1,64 @@
+.mdhui-asthma-action-plan-manager {
+    padding: 16px;
+}
+
+.mdhui-asthma-action-plan-manager-title {
```

Comment:
Can we just use a <Title> component here

Reply:
Yep, fixed.  I've also gone through and updated the other components that have a title div to use `<Title>` instead.